### PR TITLE
Added Social Signup/Login

### PR DESCRIPTION
**Pull request by yatna**
screen shot 2017-07-02 at 3 06 56 pm

Google SignIn

screen shot 2017-07-02 at 3 07 05 pm

Twitter SignIn

screen shot 2017-07-02 at 3 07 45 pm

### CODE REVIEW

**Comment by amittbhardwj** [comment]
@yatna Change Database setting: Database name to webapp and user to myuser instead of webapp3 and user2 repectively in settings.py file

**Comment by amittbhardwj** [comment]
@yatna the screenshots you posted doesn't contain login via Facebook, Github, Dropbox. Which social integration platform are you using?

**Comment by yatna** [comment]
@amittbhardwj As per my timeline , I had decided to do integrate via 2 social media platform - Google and Twitter. Although any other Social media platform can be easily added now with just a few lines of code. Currently MACC does not have accounts for facebook, github , dropbox and using my personal accounts for generating App and Secret IDs for those platforms is not a good idea.

**Comment by yatna** [comment]
@amittbhardwj Thanks for pointing out the database settings 
👍

**Comment by yatna** [comment]
@medhach Normal Login Signup Page is also present 
screen shot 2017-07-08 at 2 55 36 pm

File: webhub/views.py [hunk 1]
```diff
@@ -320,3 +322,33 @@ def details(request):
 #called when user wishes to go to the Help
 def helpPC(request):
     return HttpResponse(jinja_environ.get_template('helpPC.html').render({"pcuser":None}))  
+
+def login_real(request):
+    print("yolo")
```
Comment (amittbhardwj) [inline webhub/views.py:327]: This needs to be commented out  i think

File: webhub/views.py [hunk 2]
```diff
@@ -320,3 +322,33 @@ def details(request):
...
+
+def login_social(request):
+    username= request.POST['uname']
+    user=User.objects.get(username=username)
+    print(user.email)
+    print(username)
+    gender="Restricted"
+    location= "N.A"
+    phone="N.A"
+    num_results = Pcuser.objects.filter(user=user).count()
```
Comment (amittbhardwj) [inline webhub/views.py:338]: You can instead use get_or_create method on Pcuser

**Pull request merged**